### PR TITLE
Add Patch Update HANA Cloud example

### PR DESCRIPTION
**Pull request by kloyan**

### CODE REVIEW

File: patch-update-hana-cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+* [Deploy SAP HANA Cloud](https://developers.sap.com/tutorials/hana-cloud-deploying.html)
+* [Create Users and Manage Roles and Privileges](https://developers.sap.com/tutorials/hana-cloud-mission-trial-4.html)
+
+## How to use
+
+Import the content of [examples catalog](catalog.json) in your Automation Pilot tenant. Navigate to the *PatchUpdateHanaCloudInstance* command and trigger it.
+
+You'll need to provide values for the following input keys:
+
+* *region* - Technical name of your SAP BTP region, e.g. cf-eu10, cf-us20, cf-eu10-004
```
Comment (bisinho) [inline patch-update-hana-cloud/README.md:45]: Please , let's also add the screenshot from the account overview where the user can identify the correct region id.

**Review by bisinho — DISMISSED** [review]
Awesome work, just some very minor text adjustments.

File: patch-update-hana-cloud/README.md (hunk 1 above)
Comment (kloyan) [reply patch-update-hana-cloud/README.md:45]: Done

**Review by bisinho — APPROVED** [review]
All good , I approve it now :)

**Pull request merged**